src/modules/api.js

```diff
@@ -0,0 +1,125 @@
+import ui from './ui.js';
+import resBtn from './reservation.js';
+
+const id = 'tKVlvnEbmf4TMWB77SE7';
+const urlMealrecipe = 'https://www.themealdb.com/api/json/v1/1/lookup.php?i=';
+const urlAllMeals = 'https://www.themealdb.com/api/json/v1/1/filter.php?c=Seafood';
+const invApiUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${id}/likes`;
```

Comment:
- Persisting and retrieving record(s) with the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) well implemented! 👌